### PR TITLE
Prepare for version 2.10.3

### PR DESCRIPTION
**Pull request by tvbeek**
Type: release
Issue: -
Breaking change: no (if yes explain why)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+pdepend-2.10.3 (2022/02/23)
```
Comment (kylekatarnls) [inline CHANGELOG.md:1]:
```suggestion
pdepend-2.10.3 (2022/02/23)
==========================

```

**Review by kylekatarnls — APPROVED** [review]

**Review by AJenbo — APPROVED** [review]
``

**Comment by ravage84** [comment]
@tvbeek can you please rebase once https://github.com/pdepend/pdepend/pull/592 is merged?

**Review by ravage84 — APPROVED** [review]

**Review by AJenbo — APPROVED** [review]

**Comment by AJenbo** [comment]
Same odd server error for 5.6 tests

```
4) /home/runner/work/pdepend/pdepend/src/test/php/PDepend/Util/MathUtil/testWithBcmullFunctionDisable.phpt
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'string(7) "1000000"
+'PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20131226/pdo_sqlsrv.so' - /usr/lib/php/20131226/pdo_sqlsrv.so: cannot open shared object file: No such file or directory in Unknown on line 0
+PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20131226/sqlsrv.so' - /usr/lib/php/20131226/sqlsrv.so: cannot open shared object file: No such file or directory in Unknown on line 0
+string(7) "1000000"
 string(9) "100000000"
 string(11) "10000000000"'

phpvfscomposer:///home/runner/work/pdepend/pdepend/vendor/phpunit/phpunit/phpunit:51
```

**Comment by tvbeek** [comment]
The error was a result from the setup-php action. See: https://github.com/shivammathur/setup-php/issues/567

**Pull request merged**